manifest.json

```diff
@@ -2,16 +2,16 @@
     "spark_version": "2.2.1",
     "default_spark_dist": {
         "hadoop_version": "2.6",
-        "uri": "svt-dev.s3.amazonaws.com/spark/spark-2.2.1-bin-2.6.5-DCOS-38138.tgz"
+        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-3-SNAPSHOT-bin-2.6.tgz"
```

Comment:
This is still a `SNAPSHOT` version. Is that intentional?

Reply:
Yes I still have to merge in Susan's Hive changes before we release the spark  package. Once that's in I'll create the `2.2.1-3` tarball.

Reply:
I suppose my question is whether this is generated by the build process -- so that we don't have to manually update this each time. I'm not saying that we have to do it now, but it may make sense to file a ticket to get this working in a better way.

Reply:
That's a good point. Created ticket for that future work [DCOS-41439](https://jira.mesosphere.com/browse/DCOS-41439)